src/main/java/net/logstash/logback/appender/AbstractLogstashTcpSocketAppender.java

```diff
@@ -950,16 +957,14 @@ public synchronized void start() {
         }
 
         if (errorCount == 0) {
-
+          
             encoder.setContext(getContext());
             if (!encoder.isStarted()) {
                 encoder.start();
             }
 
-            /*
-             * Increase the core size to handle the reader thread
-             */
-            int threadPoolCoreSize = getThreadPoolCoreSize() + 1;
+            int threadPoolCoreSize = 0;
+
```

Comment:
Doesn't this need to start 1? (to account for the reader thread, which is always used).  I think that's why the original code incremented the core size by one here

Reply:
Correct. Don't know why I left it at zero... :-(